src/components/Landing.js

```diff
@@ -4,16 +4,18 @@ import '../assets/buttons.css';
 import './Landing.css';
 
 class Landing extends React.Component {
-  constructor ({ match }) {
-    super();
+  constructor (props) {
+    super(props);
     this.state = {
-      username: '',
-      period: '12month',
-      hideArtwork: false
+      username: props.username || '',
+      period: props.period || '12month',
+      hideArtwork: props.hideArtwork
     };
 
+    this.props = props;
```

Comment:
Why is that?

Reply:
To easily access the props from other places of the component, as they might add up in the future, i use it to access the `onSubmit` on line 46 instead of saving a special variable for the callback.

Reply:
`props` is an instance property on `Component` so you can use `this.props` without setting it explicitly (-:

Reply:
didn't work for me for some reason, I'll get rid of it then